From 6a3b7f6556da9157a60278ee57aac32e5b08571e Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Mon, 25 Mar 2024 11:36:33 +0100
Subject: [PATCH 1102/1141] arm64: dts: st: add SCMI power domain in
 stm32mp251.dtsi

Add protocol 17 in SCMI node, to send power-domains change state
through SCMI protocol.

Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: Ie3ea9c86cba800a6e06e32289316eb7a428b217b
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/368978
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp251.dtsi b/arch/arm64/boot/dts/st/stm32mp251.dtsi
index 75cabc6309c6..d599a79a9cdf 100644
--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -106,6 +106,11 @@ scmi: scmi {
 			#size-cells = <0>;
 			linaro,optee-channel-id = <0>;
 
+			scmi_devpd: protocol@11 {
+				reg = <0x11>;
+				#power-domain-cells = <1>;
+			};
+
 			scmi_perf: protocol@13 {
 				reg = <0x13>;
 				#clock-cells = <1>;
-- 
2.39.2

